### PR TITLE
fix: add missing permission step to ISV Debugger docs

### PR DESCRIPTION
**Pull request by lcampos**
### What does this PR do?
Adds missing step to ISV Debugger documentation

### What issues does this PR fix or reference?
@W-7939662@

### CODE REVIEW

**Review by sfsholden — APPROVED** [review]

File: docs/_articles/en/apex/isv-debugger.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ An ISV can reproduce issues in the specific environment, so problems can be diag
 
 ISV Customer Debugger is part of the Apex Interactive Debugger (`salesforcedx-vscode-apex-debugger`) extension, so you don’t need to install anything other than the Salesforce Extension Pack and its prerequisites. You can debug only sandbox orgs.
 
+1. Log into your License Management Org (LMO) and enable the **View and Debug Managed Apex** permission for your user via Profiles or Permission Sets.
```
Comment (ekapner) [inline docs/_articles/en/apex/isv-debugger.md:14]:
```suggestion
1. Log into your License Management Org (LMO) and enable the **View and Debug Managed Apex** permission for your user in Setup using Profiles or Permission Sets.
```

Comment (ekapner) [reply docs/_articles/en/apex/isv-debugger.md]: Assuming Setup and potentially multiple users. Also, assuming most dev don't have access to the LMO. So, should we say?
Contact your the License Management Org (LMO) admin to enable the View and Debug Managed Apex permission for users. (Admins will know whether they need to do this via a profile or perm set - I think we can safely assume that.)

**Review by ekapner — COMMENTED** [review]
suggested edits to step 1 - and can we assume that they will need to contact the LMO admin?

**Comment by lcampos** [comment]
> suggested edits to step 1 - and can we assume that they will need to contact the LMO admin?

@ekapner I've merged your suggestion and yes we can assume they either have admin access to the LMO or are aware that they need to contact the LMO admin.

**Review by ekapner — APPROVED** [review]

**Pull request merged**